cmd/kubectl-gadget/snapshot/process.go

```diff
@@ -41,6 +41,10 @@ func newProcessCmd() *cobra.Command {
 			},
 		}
 
+		if processGadget.Parser.GetOutputConfig().OutputMode == "tree" {
+			processGadget.printer = commonsnapshot.PrintTree
+		}
```

Comment:
Implementation is repeated in kubectl-gadget and local-gadget. I tried this https://github.com/inspektor-gadget/inspektor-gadget/commit/ec447a3957a758a9e64f111c9d3aa2bc839fcb23 to make it more generic. WDYT?